l10n/data/uk.json

```
@@ -0,0 +1,550 @@
+{
+    "bot_start_text": [
+        "👋🏼 Привітики, {}!",
+        "Цей бот використовується для отримання інформації про стан серверів та перевірки онлайну Counter-Strike: Global Offensive та багато іншого.",
+        "",
+        "Для деталей використовуйте /help."
+    ],
+    "bot_help_text": [
+        "[‎](https://telegra.ph/Detalnij-opis-08-14)ℹ️ Цей бот розроблений каналом @INCS2. Його код можна знайти за цим посиланням [GitHub](https://github.com/aquaismissing/INCS2bot).",
+        "",
+        "https://telegra.ph/Detalnij-opis-08-14"
+    ],
+    "bot_feedback_text": "💬 Будь ласка напишіть нам ваші побажання та проблеми цього бота.",
+    "bot_choose_cmd": "📖 Використовуйте одну з запропонованих команд:",
+    "bot_choose_func": "🔖 Оберіть бажану функцію:",
+    "bot_use_cancel": "Використовуйте /cancel, щоб скасувати запит.",
+    "bot_feedback_success": "Чудово! Ваше повідомлення надісланно.",
+    "bot_pmonly_text": "Ця команда працює тільки в приватних повідомленнях.",
+    "bot_back": "⏪ Назад",
+    "bot_loading": "🔄 Завантаження...",
+    "bot_author_text": "🔫 Оновления Counter-Strike",
+    "bot_author_link": "https://t.me/INCS2",
+    "bot_servers_stats": "Статистика серверів",
+    "bot_profile_info": "Інформация про користувача",
+    "bot_extras": "Додаткові функції",
+    "crosshair": "🆕 Приціл",
+    "crosshair_generate": "Створити",
+    "crosshair_decode": "Розшифрувати",
+    "crosshair_decode_example": "📖 Будь ласка, введіть код приціла (напр.: `CSGO-O4Jsi-V36wY-rTMGK-9w7qF-jQ8WB`).",
+    "crosshair_decode_error": "⚠️ Невірний код, спробуйте ще раз.",
+    "crosshair_decode_result": [
+        "🧬 Розшифровка коду приціла:",
+        "",
+        "`{}`"
+    ],
+    "currencies_usd": "Доллар США",
+    "currencies_gbp": "Британский фунт",
+    "currencies_eur": "Євро",
+    "currencies_rub": "Роzzійский рубль",
```

```suggestion
    "currencies_rub": "Россійский рубль",
```
[Лучше так](https://www.youtube.com/watch?v=stZRFhO_HGs)

---

🥇 за кринж года

---

:/